generators/server/templates/gradle/profile_dev.gradle.ejs

```diff
@@ -121,6 +185,36 @@ processResources {
 <%_ } _%>
 }
 
+task integrationTest(type: Test) {
+    minHeapSize = "128m"
+    maxHeapSize = "512m"
```

Comment:
Not quite sure it goes well with the `jvmArgs` on line 201: someone that understands how the heap of these tests plugins works to set the right values?